### PR TITLE
Initial parsing implemented

### PR DESCRIPTION
**Pull request by josephyli**
I commented out the connection for testing but you can easily uncomment it. I included the clustercfg.ini file.

To run: python cluster.py clustercfg.ini

### CODE REVIEW

File: cluster.py [hunk 1]
```diff
@@ -1,21 +1,50 @@
...
-    # connection is not autocommit by default. So you must commit to save
-    # your changes.
-    connection.commit()
-
-finally:
-    connection.close()
+configfilename = sys.argv[1]
+if os.path.isfile(configfilename):
+    # read file into a string
+    with open(configfilename) as stream:
```
Comment (TylerNakamura) [inline cluster.py]: nice! this looks really good!

Comment (josephyli) [inline cluster.py]: I would put this before the connection, since we need the config read to connect.

**Comment by josephyli** [comment]
Do I have to fork again now?

**Comment by TylerNakamura** [comment]
Good question, I don't think you do. Go ahead and try make some changes without and push to the same fork. I will let you know if you have to fork again. Big changes to the README.md on the way in 15 minutes or so.

**Pull request merged**